src/katgpucbf/utils.py

```diff
@@ -184,6 +186,76 @@ def cancel(self) -> None:
             sensor.detach(self)
 
 
+def _esterror_status(value: float) -> aiokatcp.Sensor.Status:
+    # These thresholds are thumb-sucks.
+    if 0.0 <= value < 1e-3:  # Arbitrary threshold
+        return aiokatcp.Sensor.Status.NOMINAL
+    elif value < 5e-3:  # CBF-REQ-0203 specifies 5ms
+        return aiokatcp.Sensor.Status.WARN
+    else:
+        return aiokatcp.Sensor.Status.ERROR
+
+
+def _maxerror_status(value: float) -> aiokatcp.Sensor.Status:
+    # maxerror is an over-estimate (it makes very conservative assumptions
+    # about network asymmetry and skew in the clock source). Use conservative
+    # thresholds to avoid warnings when there isn't a stratum 1 time source
+    # in the same data centre. Experimentally, maximum error is < 10ms
+    # when synchronising between Cape Town and the Karoo.
+    if 0.0 <= value < 10e-3:
+        return aiokatcp.Sensor.Status.NOMINAL
+    elif value < 0.1:
+        return aiokatcp.Sensor.Status.WARN
+    else:
+        return aiokatcp.Sensor.Status.ERROR
+
+
+def _state_status(value: aiokatcp.ClockState) -> aiokatcp.Sensor.Status:
```

Comment:
Promise I'm not being deliberately difficult, but what do you think about prepending all these `status_func`s with `_time_`? Reading `_state_status` last kinda threw me off (I know it's all self-explanatory below regarding what goes where).